global.json

```diff
@@ -1,5 +1,5 @@
 {
   "sdk": {
-    "version": "2.1.300-preview2-008324"
+    "version": "2.1.300-rc1-008673"
```

Comment:
When changin the SDK we also need to update the Contributing To Windows document + email the team notifying them that an update is needed.

Reply:
Updated